src/Common/src/Microsoft/Win32/IMarshal.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is this being added to src/Common/src?  Is it going to be consumed into projects other than System.Runtime.InteropServices?

Reply:
Even in desktop it looks like this was the only consumer.

Reply:
It really doesn't matter since it is an internal-only interface.  IMHO I'd just put it in the same namespace as the type that uses it.